deploy/template.yaml

```diff
@@ -187,7 +187,20 @@ Resources:
         ServerSideEncryptionConfiguration:
           - ServerSideEncryptionByDefault:
               SSEAlgorithm: AES256
-      BucketName: !Sub ${AWS::StackName}-test-results
+      BucketName:
+        Fn::Join:
+            - '-'
+            - - Ref: AWS::StackName
+              - test-results
+              - Fn::Select:
```

Comment:
Do we need the select here? Can we not just use the entire suffix from the stackId?

Reply:
The 12 character end of the Stack UUID is sufficient to give uniqueness of the bucket name. It is also inline with where we have used the same 12 char ID in the suffix of the `VPCTestPolicy` name

Reply:
Ah, I didn't realise that the last 12 characters were sufficient to guarantee uniqueness - if that's the case, I'm happy for this to be resolved. (For my benefit though, is there a link to documentation which says this? 😅 )

Reply:
There is some info on the format of the Stack ID [here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html#cfn-pseudo-param-stackid). It's the end of the UUID we are using here